content/articles/binance-smart-chain-tutorial/index.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /binance-smart-chain-tutorial/
+title: Getting Started With Solidity and Binance Smart Chain
+description: This tutorial will show the readers how they can get started with developing a binance smart chain token using Ethereum compatible tools such as Remix and Metamask.
+author: esther-wanjira
+date: 2022-03-08T00:00:00-21:20
+topics: [Blockchain]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/binance-smart-chain-tutorial/hero.png 
+    alt: Getting Started With Solidity and Binance Smart Chain Hero Image
+---
+Blockchain networks that support smart contracts functionality are on the rise. Smart contracts are programs that live in a blockchain and are executed on demand. In a platform like Binance, we can use the Smart chain protocol to build decentralized apps in a cross-chain network. 
+<!--more-->
+
+### Table of contents
+- [Goal](#goal)
+- [Pre-requisites](#pre-requisites)
+- [Why Binance Smart Chain?](#why-binance-smart-chain)
+- [Set Up Remix](#set-up-remix)
+- [Tokens in Binance Smart Chain](#tokens-in-binance-smart-chain)
+- [Defining Contracts and Constructor](#defining-contracts-and-constructor)
+- [Add mapping and events](#add-mapping-and-events)
+- [Create a smart contract constructor](#create-a-smart-contract-constructor)
+- [Retrieve Balances in Smart Contracts](#retrieve-balances-in-smart-contracts)
+- [Transfering Tokens Ownership](#transfering-tokens-ownership)
+- [Approve and transfer](#approve-and-transfer)
+- [A final demo in MetaMask](#a-final-demo-in-metamask)
+- [Conclusion](#conclusion)
+- [Further reading](#further-reading)
+
+### Goal
+Tokens are the building blocks of any smart contract-based blockchain. We rely on them to build exchange platforms, Dapps, and Defi platforms. This tutorial provides a context of how we can get started with developing a binance smart chain token using Ethereum compatible tools such as Remix and Metamask. In the process, we will learn the basics of smart contracts, solidity programming, tokens supply, management, and allocation.
+
+### Pre-requisites
+- Basics of [Blockchain]() technology.
+- An IDE for smart contracts. I recommend the in-browser [Remix IDE]().
+- A Metamask wallet extension. [Install here]() on your Chrome browser.
```

Comment:
Please make sure all your links are working. Also, avoid using the word "I" in your tutorial. The focus should always be on the reader. Use, "we" instead.